### ISSUE TITLE
`de.brendamour.jpasskit.signing.PKSigningUtil` is deprecated

### ISSUE THREAD

**Issue by Nhahan** [drallgood/jpasskit#508]
```java
byte[] passZipAsByteArray = PKSigningUtil.createSignedAndZippedPkPassArchive(pass, pathToTemplateDirectory, pkSigningInformation);
```

Due to the deprecation of `tutorial.md`, it's hard to know the next steps. What should I use instead?

Thank you.

**Comment by Nhahan** [comment]
Although I managed to resolve this issue by reverting to version `0.1.0`, it seems that others will still experience inconvenience.

**Comment by drallgood** [comment]
@Nhahan It is not recommended to use the old versions. they're not guaranteed to work anymore (especially when Apple makes changes to the APIs) and also contained vulnerable dependencies. 

Instead, please use the [/jpasskit/src/main/java/de/brendamour/jpasskit/signing/PKFileBasedSigningUtil.java](https://github.com/drallgood/jpasskit/blob/master/jpasskit/src/main/java/de/brendamour/jpasskit/signing/PKFileBasedSigningUtil.java) or [/jpasskit/src/main/java/de/brendamour/jpasskit/signing/PKInMemorySigningUtil.java](https://github.com/drallgood/jpasskit/blob/master/jpasskit/src/main/java/de/brendamour/jpasskit/signing/PKInMemorySigningUtil.java) or alternatively provide your own util by using the abstract [/jpasskit/src/main/java/de/brendamour/jpasskit/signing/PKAbstractSigningUtil.java](https://github.com/drallgood/jpasskit/blob/master/jpasskit/src/main/java/de/brendamour/jpasskit/signing/PKAbstractSigningUtil.java)

**Comment by drallgood** [comment]
This might also be helpful: https://github.com/drallgood/jpasskit/blob/master/jpasskit/src/test/java/de/brendamour/jpasskit/signing/PKInMemorySigningUtilTest.java#L132

**Comment by drallgood** [comment]
issue fixed with 9dac8f552a8f695df5cb966dec823c3f4fb8be90

**Comment by Nhahan** [comment]
Thank you. 😀
